hack/build-go.sh

```diff
@@ -30,5 +30,5 @@ fi
 
 mkdir -p ${BIN_PATH}
 
-echo "Building ${REPO}/cmd/${WHAT} (${VERSION})"
-CGO_ENABLED=0 GOOS=${GOOS} GOARCH=${GOARCH} go build ${GOFLAGS} -ldflags "${GLDFLAGS}" -o ${BIN_PATH}/${WHAT} ${REPO}/cmd/${WHAT}
+echo "Building ${REPO}/cmd/${WHAT} (${VERSION_OVERRIDE})"
+GO_ENABLED=0 GOOS=${GOOS} GOARCH=${GOARCH} go build ${GOFLAGS} -ldflags "${GLDFLAGS}" -o ${BIN_PATH}/${WHAT} ${REPO}/cmd/${WHAT}
```

Comment:
Was this changed intentionally...? I don't think there is a GO_ENABLED var

Reply:
Fixed and re-pushed.